minkindr/include/kindr/minimal/implementation/rotation-quaternion-inl.h

```diff
@@ -472,13 +496,24 @@ RotationQuaternionTemplate<Scalar>::log() const {
 }
 
 template<typename Scalar>
-bool RotationQuaternionTemplate<Scalar>::isValidRotationMatrix(const RotationMatrix& matrix) {
-  const Scalar kThreshold = static_cast<Scalar>(1.0e-7);
-  if (std::fabs(matrix.determinant() - static_cast<Scalar>(1.0)) > kThreshold) {
+bool RotationQuaternionTemplate<Scalar>::isValidRotationMatrix(
+    const RotationMatrix& matrix) {
+  constexpr Scalar kThreshold = static_cast<Scalar>(1.0e-8);
```

Comment:
It actually used to be 8, I changed this to 7 one or two days ago, so no code will be broken.